third_party/move/move-model/bytecode/ast-generator-tests/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+// Copyright © Aptos Foundation
```

Comment:
You also need some more arbitrary bytecode test cases.  Use .mvir or write a new assembler and use that.  But we should test some very unstructured control flow that can't be written in Move.

Reply:
I think I said above that this is a prototype and much more testing is needed. But before this can happen, we need to have the e2e test loop decompile->compile working. The team will need to maintain and extend this code. This is just the very first step.

MSIL is deprecated.